helm-charts/sep-service/templates/rbac.yaml

```diff
@@ -0,0 +1,49 @@
+#RBAC and service account
+{{- if .Values.serviceAccount  }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Values.serviceAccount.name }}
+  namespace: {{ .Release.Namespace }}
+  {{- if (.Values.serviceAccount).annotations }}
+  annotations:
+    annotations:
+    {{- with .Values.serviceAccount.annotations }}
+{{- toYaml . | nindent 4 }}
+    {{- end }}
+  {{- end }}
+---
+{{- end }}
+{{- if .Values.rbac }}
+apiVersion: v1
+kind: List
+items:
+  #define the rights
+  - apiVersion: rbac.authorization.k8s.io/v1
+    kind: Role
+    metadata:
+      name: {{ .Release.Name }}-role
+      namespace: {{ .Release.Namespace }}
+      {{- if (.Values.rbac).annotations }}
+      annotations:
+        {{- with .Values.rbac.annotations }}
+{{- toYaml . | nindent 8 }}
+        {{- end }}
+      {{- end }}
+    rules:
+{{ toYaml .Values.rbac.rules | indent 6 }} 
```

Comment:
I don't see any defaults in the values file. Could you document what RBAC/permissions does the anchor SDK require to function properly?

This change also has potentially significant security impact. Access to create role bindings is effectively privilege escalation and in secure environments and it would be better to avoid this.

Maybe the same functionality can be achieved without granting extra access?